.travis.yml

```diff
@@ -7,8 +7,8 @@ os:
 
 go:
   - tip
+  - "1.10.x"
   - 1.9.x
-  - 1.8.x
```

Comment:
Let's keep 1.8.x in the matrix for now for a few months. There are still distros that ship 1.8.x by default (sigh).